benchmarks/templates/benchmarks/landing_page.html

```diff
@@ -1,11 +1,165 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Placeholder. Actual Page Coming Soon</title>
-</head>
-<body>
-
-<h1>Placeholder. Actual Page Coming Soon</h1>
-</body>
-</html>
\ No newline at end of file
+{% load static %}
+{% load compress %}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
+<link rel="stylesheet" href="../static/benchmarks/css/landing_page.css">
+<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,300,800,700,400italic" />
+
+{# own sources #}
+{% compress js %}
+    <script defer src="{% static "/benchmarks/js/navbar.js" %}" type="text/javascript" charset="utf-8"></script>
+{% endcompress %}
+
+{% block navbar %}
+<nav class="navbar updated is-fixed-top is-transparent" role="navigation" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <div class= "container is-full-fullhd">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="https://www.brain-score.org">
+                    <img class="navbar_element image" id="navbar-logo-colored" src="/static/benchmarks/img/new_ui_elements/colored_logo.png"/>
+            </a>
+            <a role="button" class="navbar-burger" data-target="navMenu" aria-label="menu" aria-expanded="false">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+            </a>
+        </div>
+        <div id="navMenu" class="navbar-menu">
+                <div class="navbar-end">
+                    <div class="navbar-item">
+                        <button onclick="location.href='../profile'" type="button" class="button btn-1" >Log In</button>
+                        <a class="social_icon navbar-item " href="https://www.github.com/brain-score">
+                            <img class="navbar_element image" id="github_icon" src="/static/benchmarks/img/new_ui_elements/github_icon.png" />
+                        </a>
+                        <a class="social_icon navbar-item" href="https://www.twitter.com/brain_score">
+                            <img class="navbar_element image" id="twitter_icon" src="/static/benchmarks/img/new_ui_elements/twitter_icon.png"/>
+                        </a>
+                    </div>
+                </div>
+        </div>
+    </div>
+</nav>
+{% endblock %}
+
+<div class="landing container">
+    <div class="columns">
+        <div class="left_side column is-half has-text-centered-mobile">
+            <img class="logo_green image is-inline-block"  src="/static/benchmarks/img/new_ui_elements/logo_name_full.png" />
```

Comment:
```suggestion
            <img class="logo_green image is-inline-block"  src="{% static "/benchmarks/img/new_ui_elements/logo_name_full.png" %}" />
```
Use django to infer the static path, do not type yourself